### DISCUSSION TITLE
v4.9.0-rc.1

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5775] [Announcements]
Code v1.73.1

This is a pre-release for v4.9.0. 

To install via the install script, pass the `--edge` flag. 

<hr /><em>This discussion was created from the release <a href='https://github.com/coder/code-server/releases/tag/v4.9.0-rc.1'>v4.9.0-rc.1</a>.</em>

**Comment by paravatha** [comment]
@jsjoeio when are you planning to create v4.9.0? thanks in advance!

**Reply by lg-nineoneninethreenine** [reply]
Well, looking at the release patterns, there are only one release candidate occasionally. So I can say, about three days' time IST? I don't know, I think the release scheme is biweekly? Or perhaps by tomorrow, since releases seem to shoot in immediately.

@jsjoeio is there a predictable, organized release planner? I woul really like that to happen, y'know. Life ain't fun when it's predictable, but you have to be prepared, right? In this case with cronjobs.

**Reply by jsjoeio** [reply]
Apologies! I was out for a couple weeks and the team never got around to releasing it. I'm catching up on everything but hope to get to it this week or next.

@im-coder-lg not at the moment :/ We definitely need to work on that but haven't prioritized it.
